spec/rubocop/cop/rails/enum_uniqueness_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Rails::EnumUniqueness, :config do
+  subject(:cop) { described_class.new }
+
+  context 'when array syntax is used' do
+    it 'does not register an offense' do
+      inspect_source(cop, 'enum status: [ :active, :archived ]')
+
+      expect(cop.messages).to be_empty
+    end
+
+    it 'does not crash given additional enum configuration' do
```

Comment:
I'd reword this to "does not register an offense when given...". It's a bit odd to have tests testing that something doesn't crash IMO. :-)